queue_processors/queue_processor/settings.py

```diff
@@ -37,7 +37,8 @@
 # ONLY USED IN THE WebApp and should be removed in https://github.com/ISISScientificComputing/autoreduce/issues/1042
 REDUCTION_DIRECTORY = os.path.join(ARCHIVE_ROOT, 'NDX%s', 'user', 'scripts', 'autoreduction')
 # %(instrument, cycle, experiment_number, run_number)
-ARCHIVE_DIRECTORY = os.path.join(ARCHIVE_ROOT, 'NDX%s', 'Instrument', 'data', 'cycle_%s', 'autoreduced', '%s', '%s')
+CYCLE_DIRECTORY = os.path.join(ARCHIVE_ROOT, 'NDX%s', 'Instrument', 'data', 'cycle_%s')
+ARCHIVE_DIRECTORY = os.path.join(CYCLE_DIRECTORY, 'autoreduced', '%s', '%s')
```

Comment:
I can't find this to actually be used anywhere. Same for `TEST_REDUCTION_DIRECTORY` and `TEST_ARCHIVE_DIRECTORY`. Perhaps we can just remove them?